### PR TITLE
Adds limb ashing and limb necrosis

### PR DESCRIPTION
**Pull request by MadmanMartian**
Limb ashing occurs if the burn damage received at an instant is greater than 30, this is a near-mirror of limb gibbing from high brute damage.

Limb necrosis occurs should a limb acquire too much burn damage, this causes the flesh to rapidly melt off, creating a dead limb. This is a near-mirror of limb bones breaking from too much brute damage.

Moved limb breaking from limb process, to on damage. Odd that it needs somebody to be alive for their limbs to break.

:cl:
 * rscadd: Adds limb necrosis, where the flesh of the limb rapidly melts off. This occurs from increased amount of burn damage to a limb. Treatment is currently through limb replacement or cloning
 * rscadd: Adds limb ashing, which occurs when a tremendous amount of burn damage is done to a human limb at once.

### CODE REVIEW

File: code/modules/organs/organ_external.dm [hunk 1]
```diff
@@ -214,6 +224,11 @@
 					droplimb(1)
 					return
 
+				//burn
+				if(burn > 30 && prob(3 * brute))
```
Comment (DrSnips) [inline code/modules/organs/organ_external.dm:228]: Why is is checking for brute and burn if its just burn related?

Comment (MadmanMartian) [reply code/modules/organs/organ_external.dm:228]: oh shit, good catch

**Comment by Blithering** [comment]
Does necrosis work like in the vampire disease version? And is it incurable without limb transplants if so?

**Comment by MadmanMartian** [comment]
It is similar in name alone, as I could not think of a better term for it

**Comment by crudequip** [comment]
"slough" is a valid medical term that essentially means the same thing

**Comment by Blithering** [comment]
Or nerve damage in general, or you could use first, second, third degree burns

**Comment by arc166** [comment]
So diona now perma lose limbs in 1 shot from cap laser? Maybe regular too, I forget. 
Also why do you specifically state 30 for ashing then ''too much for necrosis''. Give solid numbers for both.

**Comment by MadmanMartian** [comment]
I shot a diona with the captains laser in the left leg. First shot did nothing, second shot necrotized, third shot didn't ash, fourth shot he was a husk.

There's a lot more issues for diona than losing a leg to a laser.

**Comment by utilitas-** [comment]
I strongly feel that nothing should be a mirror of any other damage type, otherwise we might as well merge them and simplify them. Bump the values for ashing and necrosis a little higher, especially since necrosis sounds like a much bigger deal than a bone fracture. Plus, necrosis shouldn't arrive immediately, but first arrive through blistering/boiling that then turns to necrotising tissue if not treated with ointment.

**Comment by MadmanMartian** [comment]
I can't go much higher as there's a maximum cap on damage (limb dependent, think it's 75 for an arm for example)

But that necrosis idea is pretty good. I'll change it to that

**Comment by agentofzog2** [comment]
Why do lasers need a buff again?

**Comment by MadmanMartian** [comment]
mcfo

**Comment by agentofzog2** [comment]
Ok but why though?

**Comment by ghost** [comment]
> 
> 
> Ok but why though?

How many laser weapons are there that do over 30 damage in one shot?

**Comment by agentofzog2** [comment]
> 
> 
> > Ok but why though?
> 
> How many laser weapons are there that do over 30 damage in one shot?

A couple but necrosis sounds like it applies to all burn damage.

**Comment by ghost** [comment]
What if there was a way to reverse it? It's already less worse than before in that accumulating laser shots on your limbs will only make you lose the limb instead of just dying when total damage is over 200.

**Comment by Intigracy** [comment]
Aren't you the one who just said that the limb damage PR nerfed everything that's not a stun.

Now you're asking why buff lasers.

😂😂😂😂

**Comment by agentofzog2** [comment]
> 
> 
> Aren't you the one who just said that the limb damage PR nerfed everything that's not a stun.
> 
> Now you're asking why buff lasers.
> 
> joyjoyjoyjoy

Because lasers are still great, especially when paired with a stun.

**Comment by MadmanMartian** [comment]
From Tokanova:

A method of treating it could be rewiring the lost veins using a veinlayer, then applying syinthetic flesh to renew the lost flesh.

I'll be adding this tomorrow.

**Comment by SonixApache** [comment]
😔

**Comment by Intigracy** [comment]
😔

**Comment by vampirebat74** [comment]
f

**Pull request closed without merging**